frontend/src/app/AppLayout/AppLayout.tsx

```diff
@@ -13,12 +13,15 @@ import { routes } from "../routes";
 import packitLogo from "../../static/logo.png";
 import { useState, useEffect } from "react";
 
-const AppLayout = () => {
+const AppLayout: React.FC = () => {
```

Comment:
Will probably change this back, it's not needed. `React.FC` or `React.FunctionComponent` helps with declaring props to the component due to generics. So if there are props we use `React.FC<InterfaceOfPropsName>` and it will be correctly applied to the parameter for the props
```suggestion
const AppLayout = () => {
```